pandas/tests/series/methods/test_value_counts.py

```diff
@@ -179,6 +179,13 @@ def test_value_counts_categorical_with_nan(self):
             res = ser.value_counts(dropna=False, sort=False)
             tm.assert_series_equal(res, exp)
 
+    def test_value_counts_interval_bins(self):
```

Comment:
pls add a comment e.g `# GH#33652 case where [...]`

Reply:
I fixed this and above issues.

Reply:
i meant add the comment on L183 inside the the test